### PR TITLE
0x05c - Update Angr Example

### PR DESCRIPTION
**Pull request by kousha1999**
Angr exploit updated. It now works with Angr 9.2.1 correctly.

### CODE REVIEW

**Comment by cpholguera** [comment]
Thanks a lot for the update @kousha1999, we'll take a look!

File: Document/0x05c-Reverse-Engineering-and-Tampering.md [hunk 1]
```diff
@@ -1217,17 +1217,20 @@ The final solution script is presented below:
 
 ```python
 import angr
```
Comment (cpholguera) [inline Document/0x05c-Reverse-Engineering-and-Tampering.md:1219]:
```suggestion
import angr # Version: 9.2.2
```

File: Document/0x05c-Reverse-Engineering-and-Tampering.md [hunk 2]
```diff
@@ -1257,13 +1260,14 @@ Also, it may appear as if the script is simply reading the solution string from
...
-WARNING | cle.loader | The main binary is a position-independent executable.
-It is being loaded with a base address of 0x400000.
+$ python3 solve.py
+WARNING | ... | cle.loader | The main binary is a position-independent executable. It is being loaded with a base address of 0x400000.
 
-b'ABGAATYAJQAFUABB'
+b'JACE6ACIARNAAIIA'
 ```
 
+Now you can run the validate binary in your Android device to verify the solution as indicated [here](../Crackmes/README.md#android-license-validatorandroidlicense01-"android-license-validator").
```
Comment (cpholguera) [inline Document/0x05c-Reverse-Engineering-and-Tampering.md:1269]:
```suggestion
Now you can run the validate binary in your Android device to verify the solution as indicated [here](../Crackmes/README.md#android-license-validator).
```

**Review by cpholguera — APPROVED** [review]
This is great @kousha1999, thank you so much for keeping the MSTG up to date!

**Comment by cpholguera** [comment]
FYI I updated the version again (originally 9.2.1 in your PR) since it worked fine with 9.2.2. Thank you!

**Comment by kousha1999** [comment]
> This is great @kousha1999, thank you so much for keeping the MSTG up to date!

It's my pleasure.

**Pull request merged**